.eslintrc

```diff
@@ -17,6 +17,9 @@
     },
     {
       "files": ["**/*.tsx"],
+      "globals": {
```

Comment:
We don't need this, please remove 🙇

Reply:
https://github.com/nodejs/nodejs.org/pull/5244#issuecomment-1503663833

Reply:
Sorry, but that is no argument. Please remove this from your PR, this is an unrelated change.

Reply:
Maybe your ESLint cache is bad. I just pulled the branch and ran `npm run lint:js` after `npm ci` and there is no error

Reply:
His branch doesn't use JSX types, he was trying to incorporate this "general" fix on his branch, hence why I said these are unrelated changes.

Reply:
@nschonni the error is not present in the existing branch, it appeared when we tried to migrate components from nodejs.dev. I believe removing this line will throw an error in CI, we will try an alternative fix.

Reply:
Removing this line will not cause an issue once this PR gets merged. And I know I am repeating myself, but I already mentioned that I'm not letting the JSX global PR land. Not in the way it is.

I honestly believe the PR introducing the JSX.Element type usages should remove that explicit JSX type, and that's it.